### DISCUSSION TITLE
音箱没有media_player实体

### DISCUSSION THREAD

**Discussion by anxms** [XiaoMi/ha_xiaomi_home#82] [Ideas]
音箱没有media_player实体，生成的都是按钮实体，希望增加

**Comment by vanscer** [comment]
![image](https://github.com/user-attachments/assets/ef1a6b8e-bd40-4ca3-a421-c7c44230d554)

**Comment by JimmyKmi** [comment]
参考：https://note.p.lzj.ac.cn/s/bd635048-ae38-43c9-9f31-d61175a01cf6/doc/6k65ash5bge5ocn57y65asx-S3aYBqlRvw

**Comment by SusanPhevos** [comment]
v0.4.0 已支持
